elastic/kibana
# [Actions] Taking space id into account when creating email footer link

Resolves https://github.com/elastic/kibana/issues/99300

## Summary

Passing in `spaceId` to the function that creates the footer link. If `spaceId` is not default, prepends `/s/${spaceId}` to the footer link.

## To Verify

Create a rule in the default space and in another space. Add an email action to both rules. Verify that the email received has a footer link that correctly resolves to the rule details page, including space id if required.

### Checklist

Delete any items that are not applicable to this PR.
